tests/security/ebpf/check_cap_bpf.pm

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'CAP_BPF' capability is available:
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#103932, tc#1769831
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $f_tcpdump = '/usr/sbin/tcpdump';
+    my $capability = 'cap_bpf';
+
+    select_console 'root-console';
+
+    # Install packages
+    zypper_call('in libcap-progs tcpdump');
+
+    assert_script_run("getcap $f_tcpdump");
+    assert_script_run("setcap $capability+eip $f_tcpdump");
```

Comment:
I don't really understand the comman, do you want run 
`#setcap cap_bpf+eip /usr/sbin/tcpdump` 
if yes, what's the meaning you need to set a variable here?

Reply:
yes, want to check the capability "cap_bpf" is available for user commands.
You can man "getcap/setcap/capabilities" for more info on this area.

Reply:
IMO, the code is really simple.

Reply:
Thank you for the explanation offline. No more comments. LGTM.